src/PhpDocument.php

```diff
@@ -176,7 +176,7 @@ public function parse()
         $diagnostics = [];
         foreach ($errors as $error) {
             $diagnostic = new Diagnostic();
-            $startLine = $error->getStartLine() === -1 ? 0 : $error->getStartLine() - 1;
+            $startLine = max($error->getStartLine() - 1, 0);
             $startColumn = $error->hasColumnInfo() ? $error->getStartColumn($this->content) - 1 : 0;
             $endLine = $error->getEndLine() === -1 ? $startLine : $error->getEndLine() - 1;
```

Comment:
I wondered about the end line too... In this patch I just wanted to protect from `-1` line values. Your suggestion is to protect `endLine` being lower than `startLine`. Which is logical, but I am not sure if we would not cause some side effect in edge cases.